samples/core/blur/blur.hpp

```diff
@@ -41,6 +41,9 @@ class BlurCppExample {
     // Query device and runtime capabilities
     std::tuple<bool, bool, bool> query_capabilities();
 
+    // Query device support for OpenCL 2.0
+    bool query_opencl_2_0_support();
```

Comment:
See comment above - I think this is more about OpenCL C 2.0 support, not OpenCL 2.0 support?